include/cuda/std/detail/libcxx/include/experimental/__p0009_bits/standard_layout_static_array.hpp

```diff
@@ -53,12 +53,16 @@
 #  include "no_unique_address.hpp"
 #endif
 
-#include <cuda/std/array>
+#ifndef __cuda_std__
+#include <array>
+#endif
 #ifdef __cpp_lib_span
 #include <span>
```

Comment:
Do we also need to protect `#include <span>` with `#ifndef __cuda_std__` ... `#endif` ?

Reply:
We don't have a `libcudacxx` version of `span` (or I didn't find one).

Reply:
@youyu3 Oh, oof : - (  Should we file an issue to add one?  We would need it for mdspan just for a few things that take `span` as input.

Reply:
Now we have and it worked for my initial tests.  More tests to be added.